libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx

```diff
@@ -88,6 +89,13 @@ export interface IAttributeFilterDropdownButtonProps {
      */
     icon?: ReactNode;
 
+    /**
+     * Customize attribute filter tooltip component.
```

Comment:
detail: "customize content of the"... (as it does not customize the tooltip itself)

Reply:
Updated.